postgres/datadog_checks/postgres/metadata.py

```diff
@@ -0,0 +1,122 @@
+# (C) Datadog, Inc. 2021-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+from typing import Dict, Optional, Tuple  # noqa: F401
+
+import time
+import json
+import psycopg2
+
+try:
+    import datadog_agent
+except ImportError:
+    from ..stubs import datadog_agent
+
+from datadog_checks.base import is_affirmative
+from datadog_checks.base.utils.db.utils import (
+    DBMAsyncJob,
+    RateLimitingTTLCache,
+    default_json_event_encoding,
+    obfuscate_sql_with_metadata,
+)
+from datadog_checks.postgres.connections import MultiDatabaseConnectionPool
+from datadog_checks.base.utils.tracking import tracked_method
+
+# default collection interval in minutes
+DEFAULT_COLLECTION_INTERVAL = 30
```

Comment:
I know this is not frequently updating data, but 30 minutes is so long.  Imagine looking at a 1-hour graph and having no idea where the change actually happened. 10m is the minimum I would declare for any data we send unless the query is very expensive. I think 5m would also be reasonable.